tox.ini

```diff
@@ -0,0 +1,23 @@
+[tox]
+envlist=
+    py27,
+    py34,
+    py35
+
+[flake8]
+max-line-length= 100
+exclude= tests/*
+
+[testenv]
+commands=py.test --tb native {posargs:tests}
```

Comment:
The `tests` path here isn't correct for this repo: it should probably either be `.` or `examples/cyberdyne/test`?